src/Azure/Azure.Quantum.Client/SubmitterFactory.cs

```diff
@@ -35,7 +35,7 @@ public static class SubmitterFactory
                 ImmutableArray.Create("FullComputation")),
             new SubmitterInfo(
                 new Regex(@"\Aquantinuum\.([\w-_]+\.)*[\w-_]+\z"),
-                "Microsoft.Quantum.Providers.Quantinuum.Targets.QuantinuumQirSubmitter, Microsoft.Quantum.Providers.Honeywell",
+                "Microsoft.Quantum.Providers.Honeywell.Targets.QuantinuumQirSubmitter, Microsoft.Quantum.Providers.Honeywell",
```

Comment:
Same than this [comment](https://github.com/microsoft/qsharp-runtime/pull/1099/files#r998664553) for this case.

Reply:
done